clients/cobol-lsp-vscode-extension/src/constants.ts

```diff
@@ -20,6 +20,7 @@ export const SETTINGS_CPY_FILE_ENCODING: string =
   "cobol-lsp.cpy-manager.copybook-file-encoding";
 export const SETTINGS_SQL_BACKEND: string = "cobol-lsp.target-sql-backend";
 export const SETTINGS_DIALECT = "cobol-lsp.dialects";
+export const COMPILE_OPTIONS = "cobol-lsp.compiler.options";
```

Comment:
```suggestion
export const SETTINGS_COMPILE_OPTIONS = "cobol-lsp.compiler.options";
```
Don't forget to update ts tests for pg